Tests/GarageStorageTests/MappableObjectTests.swift

```diff
@@ -364,4 +364,25 @@ class MappableObjectTests: XCTestCase {
             XCTAssertEqual(sam?.birthdate.timeIntervalSinceReferenceDate ?? 0, -1609459200.0, "Reconstituted date failed")
         }
     }
+    
+    func testNonExistentObject() {
+        let garage = Garage()
+        
+        // Swift behavior: be able to return nil for not found, not throw an error.
+        do {
+            let frodo = try garage.retrieveObject(ObjCPerson.self, identifier: "Frodo")
+            XCTAssertNil(frodo, "Should be nil")
+        }
+        catch {
+            XCTFail("Should not have thrown an error: \(error)")
+        }
+        
+        // Objective-C behavior: throw an error so that the ObjC runtime can return nil to an ObjC caller.
+        do {
```

Comment:
Observation, then question: It would be "tighter" to declare `__retrieveObjectObjC` as obsolete in Swift, and then (ugh) rewrite this bit of unit testing as a separate unit test of `retrieveObjectOfClass` _implemented in Obj-C_, rather than Swift.

So, question: worth the effort? Or too paranoid?